### ISSUE TITLE
[BUG] Rendering incompatability with mods adding custom shafts.

### ISSUE THREAD

**Issue by DaRealSparky** [Alastors/Extended-Cogwheels-Unofficial#6]
**Describe the bug**
In reference to [Issue #83](https://github.com/Rabbitminers/Extended-Cogwheels/issues/83#issue-2061010978) on the official Extended Cogwheels Repo. Mods that add custom shafts and other blocks using `BracketedKineticBlockEntityRenderer` are rendered as Shaftless Cogwheels in game. There is another issue in the post mentioning Z-fighting, but I have yet to see any in my own testing.

**To Reproduce**
1: Load Minecraft with Create, Create Extended Cogwheels (Official or Unofficial), and either [Create Connected](https://www.curseforge.com/minecraft/mc-mods/create-connected) or [Create encased](https://www.curseforge.com/minecraft/mc-mods/create-encased) or both.
2: Load creative world
3: Place Create Connected Shear Pin or custom shaft from Create Encased.
4: Observe strange looking "shaft."

**Expected behavior**
All blocks are rendered as they should be.

**Screenshots**
Refer to linked issue.

**Loader & Other Mods:**
 - Forge
 - Create Connected, Create Encased, and a load more Create addons. List can be provided if required or requested.

**Additional context**
This has been an issue with Extended Cogwheels for quite a while now.

**Comment by Alastors** [comment]
> **Describe the bug** In reference to [Issue #83](https://github.com/Rabbitminers/Extended-Cogwheels/issues/83#issue-2061010978) on the official Extended Cogwheels Repo. Mods that add custom shafts and other blocks using `BracketedKineticBlockEntityRenderer` are rendered as Shaftless Cogwheels in game. There is another issue in the post mentioning Z-fighting, but I have yet to see any in my own testing.
> 
> **To Reproduce** Steps to reproduce the behavior: 1: Load Minecraft with Create, Create Extended Cogwheels (Official or Unofficial), and either [Create Connected](https://www.curseforge.com/minecraft/mc-mods/create-connected) or [Create encased](https://www.curseforge.com/minecraft/mc-mods/create-encased) or both. 2: Load creative world 3: Place Create Connected Shear Pin or custom shaft from Create Encased. 4: Observe strange looking "shaft."
> 
> **Expected behavior** All blocks are rendered as they should be.
> 
> **Screenshots** Refer to linked issue.
> 
> **Loader & Other Mods:**
> 
>     * Forge
> 
>     * Create Connected, Create Encased, and a load more Create addons. List can be provided if required or requested.
> 
> 
> **Additional context** This has been an issue with Extended Cogwheels for quite a while now.

oh thanks, I'll look into it, sorry for getting to this so late, I've been taking a break from modding for some personal issues.

**Comment by DaRealSparky** [comment]
No worries.

**Comment by Alastors** [comment]
Yeah so started looking into this, I may be looking into simplifying the rendering on this end. The latest version of create totally killed the mixins this mod was using + it seems like some of the special stuff related to it may be causing a bit too much overhead, especially in relation to things like how the brackets don't show up either.

**Comment by Alastors** [comment]
Yeah I think I'll go with the suggestion from connected and drop mixin support for a more natural extension system.

It shouldn't be a major challenge to disrupt the texture itself, but we'll see. 

I've done something like this back in the day so it shouldn't be too bad

**Comment by Alastors** [comment]
<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/9774b776-d17d-4bd2-9fbb-4e0ff7ce59f6" />

Took 4 days, but I finally fixed it
https://github.com/Rabbitminers/Extended-Cogwheels/issues/83#issue-2061010978

https://github.com/petrolpark/Petrols-Parts/issues/8

Working on the encased cogwheels next

**Comment by Alastors** [comment]
https://github.com/user-attachments/assets/af381cba-f8d8-4e60-9830-771af1b7393e

Fixed the remaining incompatibility issue as well.

**Comment by Alastors** [comment]
Fixed: https://github.com/hlysine/create_connected/issues/90
